kpi/views/v2/data.py

```diff
@@ -217,6 +217,43 @@ class DataViewSet(AssetNestedObjectViewsetMixin, NestedViewSetMixin,
     >        }
 
 
+    ### Bulk updating of submissions
+
+    <pre class="prettyprint">
+    <b>PATCH</b> /api/v2/assets/<code>{uid}</code>/data/bulk/
+    </pre>
+
+    > Example
+    >
+    >       curl -X PATCH https://[kpi]/api/v2/assets/aSAvYreNzVEkrWg5Gdcvg/data/bulk/
+
+    > **Payload**
+    >
+    >        {
+    >           "submission_ids": [{integer}],
+    >           <field_to_update_1>: <value_1>,
+    >           <field_to_update_2>: <value_2>,
+    >           <field_to_update_n>: <value_n>
+    >        }
+
+    where `<field_to_update_n>` is a string and should be an existing XML field value of the submissions.
+    If `<field_to_update_n>` is part of a group or nested group, the field must follow the group hierarchy
+    structure, i.e.:
+
+    If the field is within a group called `group_1`, the field name is `question_1` and the new value is `new value`,
+    the payload should contain an item with the following structure:
+
+    <pre class="prettyprint">
+    "group_1/question_1": "new value"
+    </pre>
+
+    Similarly, if there are `N` nested groups, the structure will be:
+
+    <pre class="prettyprint">
+    "group_1/sub_group_1/.../sub_group_n/question_1": "new value"
+    </pre>
+
+
```

Comment:
Thanks for including detailed documentation and a detailed description of the pull request. In the PR description, could you leave your complete text intact but move the `## Description` heading below it, and then write a short (few sentences tops) description beneath that heading that's suitable for inclusion in our release notes ([example](https://community.kobotoolbox.org/t/release-notes-version-2-020-25/9473))? Basically:
```
As much as you want to say, in as much detail as you'd like. This is a great PR
and here are several reasons why! Let me explain in useful detail how you might
take advantage of this PR as a user and understand how it works as a developer.
This text will not be included in the release notes, but the release notes will have
a hyperlink to this PR so that readers can easily access the full details.

## Description
Added great feature to do great stuff.

## Related issues
Closes #xyz, etc. This also appears in the release notes, in the rightmost column.
No changes are needed from what you already have.
```

Reply:
I'll do that 🕺 I read [this article](https://mtlynch.io/code-review-love/) while working on this feature and I was inspired to make this documentation a bit more extensive than I have in the past — I maybe got a bit carried away 🙃

Reply:
lol: "But I don’t want my reviewer to fall in love with me"
i'll have to read the article :) the documentation you wrote is great, and i do indeed love it. it just won't fit in our silly markdown table that we post to Discourse. i'm very glad that it will remain here in GitHub for future reference, though.
not to dampen the spirits of your new description for the release notes (with "shiny new feature," a phrase that surely makes Tino proud!), but i may edit it to clarify that this is API only—just in case "Backend support" in the title doesn't make it sufficiently clear for non-developer folks.